### PR TITLE
11.0 mig base partner merge

### PR DESCRIPTION
**Pull request by JayVora-SerpentCS**
Module Migrated to 11.0

### CODE REVIEW

**Review by tschanzt — CHANGES_REQUESTED** [review]
Only minor stuff:
  - The utf-8 header isn't needed anymore for python3. 
  - Remove the .pot file, since they aren't stored in the module anymore.

Otherwise i tested it and it works. :+1:

**Review by yvaucher — APPROVED** [review]
Appart from @tschanzt remarks, LGTM

File: base_partner_merge/README.rst [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+Images
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
+
+Author
+------
+
+Yannick Vaucher
```
Comment (simahawk) [inline base_partner_merge/README.rst]: add email `<yannick.vaucher@camptocamp.com>` and "Do not contact contributors directly about support or help with technical issues." see https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

File: base_partner_merge/__init__.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
```
Comment (simahawk) [inline base_partner_merge/__init__.py]: remove coding headers

File: base_partner_merge/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+# Copyright 2016 Camptocamp SA
+# Copyright 2017 Jarsa Sistemas
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    'name': "Deduplicate Contacts (No CRM)",
+    'version': '11.0.1.0.0',
+    'summary': "Partner merge wizard without dependency on CRM",
+    'category': 'Generic Modules/Base',
+    'author': "Camptocamp,Odoo Community Association (OCA)",
+    'license': 'AGPL-3',
```
Comment (simahawk) [inline base_partner_merge/__manifest__.py]: add `'website': 'https://github.com/OCA/partner-contact',`

File: base_partner_merge/views/base_partner_merge.xml [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+    <record model="ir.actions.act_window" id="action_partner_deduplicate">
+        <field name="name">Deduplicate Contacts</field>
+        <field name="res_model">base.partner.merge.automatic.wizard</field>
+        <field name="view_type">form</field>
+        <field name="view_mode">form</field>
+        <field name="target">new</field>
+        <field name="context">{'active_test': False}</field>
+    </record>
+
+    <record model='ir.ui.view' id='base_partner_merge_automatic_wizard_form'>
```
Comment (simahawk) [inline base_partner_merge/views/base_partner_merge.xml]: pls do not mixup quotes: use double quotes all around

File: base_partner_merge/views/base_partner_merge.xml [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+        <field name='model'>base.partner.merge.automatic.wizard</field>
+        <field name='arch' type='xml'>
+            <form string='Automatic Merge Wizard'>
+                <sheet>
+                    <group attrs="{'invisible': [('state', '!=', 'finished')]}" col="1">
+                        <h2>There is no more contacts to merge for this request...</h2>
+                        <button name="%(action_partner_deduplicate)d" string="Deduplicate the other Contacts" class="oe_highlight"
+                        type="action"/>
+                    </group>
+                    <p class="oe_grey" attrs="{'invisible': [('state', '!=', ('option'))]}">
```
Comment (simahawk) [inline base_partner_merge/views/base_partner_merge.xml]: do not wrap `option` w/ parenthesis: `[('state', '!=', 'option')]`

File: base_partner_merge/views/base_partner_merge.xml [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+        <field name="context">{'active_test': False}</field>
+    </record>
+
+    <record model='ir.ui.view' id='base_partner_merge_automatic_wizard_form'>
+        <field name='name'>base.partner.merge.automatic.wizard.form</field>
+        <field name='model'>base.partner.merge.automatic.wizard</field>
+        <field name='arch' type='xml'>
+            <form string='Automatic Merge Wizard'>
+                <sheet>
+                    <group attrs="{'invisible': [('state', '!=', 'finished')]}" col="1">
```
Comment (simahawk) [inline base_partner_merge/views/base_partner_merge.xml]: add a name to this group

File: base_partner_merge/views/base_partner_merge.xml [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+                        <button name="%(action_partner_deduplicate)d" string="Deduplicate the other Contacts" class="oe_highlight"
+                        type="action"/>
+                    </group>
+                    <p class="oe_grey" attrs="{'invisible': [('state', '!=', ('option'))]}">
+                        Select the list of fields used to search for
+                        duplicated records. If you select several fields,
+                        Odoo will propose you to merge only those having
+                        all these fields in common. (not one of the fields).
+                    </p>
+                    <group attrs="{'invisible': ['|', ('state', 'not in', ('selection', 'finished')), ('number_group', '=', 0)]}">
```
Comment (simahawk) [inline base_partner_merge/views/base_partner_merge.xml]: same here and all the other groups

File: base_partner_merge/wizard/base_partner_merge.py [hunk 8]
```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Camptocamp SA
+# Copyright 2017 Jarsa Sistemas
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo.addons.crm.wizard.base_partner_merge import *  # noqa
```
Comment (simahawk) [inline base_partner_merge/wizard/base_partner_merge.py]: seems we import only 2 things: `MergePartnerLine` and `MergePartnerAutomatic` -> remove the wildcard

**Review by simahawk — CHANGES_REQUESTED** [review]

**Comment by simahawk** [comment]
@JayVora-SerpentCS  ping :)

**Comment by JayVora-SerpentCS** [comment]
checking

**Review by huyly0909 — APPROVED** [review]

**Comment by yvaucher** [comment]
LGTM, I'll merge and squash the last commits in one.

**Comment by yvaucher** [comment]
Merged at https://github.com/OCA/partner-contact/commit/66d236fe99425b4e0ccdbb398ec7aac21fbea159

**Comment by yvaucher** [comment]
Thank you @JayVora-SerpentCS

**Pull request closed without merging**